src/openklant/utils/tests/test_validators.py

```diff
@@ -74,11 +74,20 @@ def test_validate_postal_code(self):
         self.assertIsNone(validate_postal_code("1015 cJ"))
 
     def test_validate_phone_number(self):
+        valid_phone_numbers = [
```

Comment:
Could you add the numbers here that are [mentioned here](https://github.com/maykinmedia/open-klant/issues/235#issuecomment-2323875962) (the top 5) to verify they work as expected?